packages/grid/x-data-grid/src/locales/ukUA.ts

```diff
@@ -94,33 +94,33 @@ const ukUAGrid: Partial<GridLocaleText> = {
   filterOperatorIsEmpty: 'порожній',
   filterOperatorIsNotEmpty: 'не порожній',
   filterOperatorIsAnyOf: 'будь-що із',
-  // 'filterOperator=': '=',
-  // 'filterOperator!=': '!=',
-  // 'filterOperator>': '>',
-  // 'filterOperator>=': '>=',
-  // 'filterOperator<': '<',
-  // 'filterOperator<=': '<=',
+  'filterOperator=': '=',
+  'filterOperator!=': '!=',
+  'filterOperator>': '>',
+  'filterOperator>=': '>=',
+  'filterOperator<': '<',
+  'filterOperator<=': '<=',
 
   // Header filter operators text
-  // headerFilterOperatorContains: 'Contains',
-  // headerFilterOperatorEquals: 'Equals',
-  // headerFilterOperatorStartsWith: 'Starts with',
-  // headerFilterOperatorEndsWith: 'Ends with',
-  // headerFilterOperatorIs: 'Is',
-  // headerFilterOperatorNot: 'Is not',
-  // headerFilterOperatorAfter: 'Is after',
-  // headerFilterOperatorOnOrAfter: 'Is on or after',
-  // headerFilterOperatorBefore: 'Is before',
-  // headerFilterOperatorOnOrBefore: 'Is on or before',
-  // headerFilterOperatorIsEmpty: 'Is empty',
-  // headerFilterOperatorIsNotEmpty: 'Is not empty',
-  // headerFilterOperatorIsAnyOf: 'Is any of',
-  // 'headerFilterOperator=': 'Equals',
-  // 'headerFilterOperator!=': 'Not equals',
-  // 'headerFilterOperator>': 'Greater than',
-  // 'headerFilterOperator>=': 'Greater than or equal to',
-  // 'headerFilterOperator<': 'Less than',
-  // 'headerFilterOperator<=': 'Less than or equal to',
+  headerFilterOperatorContains: 'Містить',
+  headerFilterOperatorEquals: 'Дорівнює',
+  headerFilterOperatorStartsWith: 'Починається з',
+  headerFilterOperatorEndsWith: 'Закінчується на',
+  headerFilterOperatorIs: 'Дорівнює',
+  headerFilterOperatorNot: 'Не дорівнює',
+  headerFilterOperatorAfter: 'Більше ніж',
+  headerFilterOperatorOnOrAfter: 'Більше або дорвінює',
+  headerFilterOperatorBefore: 'Менше ніж',
+  headerFilterOperatorOnOrBefore: 'Менше або дорівнює',
```

Comment:
These are labels for date operators, shouldn't it be like this?

```suggestion
  headerFilterOperatorAfter: 'Після',
  headerFilterOperatorOnOrAfter: 'Після (включно)',
  headerFilterOperatorBefore: 'Раніше',
  headerFilterOperatorOnOrBefore: 'Раніше (включно)',
```

Reply:
Звучить цілком логічно, але для орієнтування взяв переклади, що були вище, щоб була однорідність, а змінювати попереднє не став.

90 рядок:
```javascript
  filterOperatorAfter: 'більше ніж',
  filterOperatorOnOrAfter: 'більше або дорівнює',
  filterOperatorBefore: 'менше ніж',
  filterOperatorOnOrBefore: 'менше або дорівнює',
  ```
  Якщо змінювати ці, чи варто змінювати ті, що вже були?

Reply:
переглянув декілька мов, де є переклади обох - в деяких ідентичні рядки (німецька, французька) в інших відрізняються (іспанська).